src/common/BaseCommand.ts

```diff
@@ -55,8 +55,18 @@ export abstract class BaseCommand
                 return Logger.child(this.commandName);
             })
             .then((logger) => {
+                // extract the log level flag (if any) and
+                // set the logger's level to this value
+                const logLevelFlag: string | undefined = (
```

Comment:
We were never actually using the value passed in for the `loglevel` flag (even in the past). Now that I'm bringing in these flags for backwards compatibility with SFDX VSCode plugin, I'm going to go ahead and actually wire it up too.

Reply:
Can you help me remember why we took the out the flags please?

Reply:
We didn't take out the flags. The older versions of `SfCommand` used to have these built-in but the new versions don't have these built-in anymore (not sure why though, and that's a different discussion).

It is now the implementors job to add whatever flags they want. I learned this when I was trying to launch commands from VSCode Extension using our latest builds of `lwc-dev-*` and saw errors on undefined `--json` and `--loglevel` flags.